src/omero_zarr/raw_pixels.py

```diff
@@ -185,11 +185,13 @@ def add_raw_image(
 
                     if (level + 1) < level_count:
                         # resize for next level...
-                        plane = cv2.resize(
+                        plane = resize(
                             plane,
-                            dsize=(size_x // 2, size_y // 2),
-                            interpolation=cv2.INTER_NEAREST,
-                        )
+                            output_shape=(size_x // 2, size_y // 2),
```

Comment:
shouldn't the x and y be swapped here? If the shape is `(y, x)` then we want `output_shape=(size_y // 2, size_x // 2)`
I think this was already a bug in the previous `cv2` code.

Reply:
Thanks for spotting this. Fixed in f8cc648